packages/codegen-ui-react/lib/__tests__/react-component-render-helper.test.ts

```diff
@@ -345,4 +346,15 @@ describe('react-component-render-helper', () => {
       assertASTMatchesSnapshot(exp);
     });
   });
+
+  describe('parseNumberOperand', () => {
+    test('should parse int if field data type is Int', () => {
+      expect(parseNumberOperand('10', { dataType: 'Int', readOnly: false, required: false, isArray: false })).toBe(10);
+    });
+    test('should parse int if field data type is Int', () => {
```

Comment:
nit: the test name should change to `Float`

Reply:
ah damn haha. ill fix it in another pr